ARSIS-Telemetry-GroundControl/telemetry-api/app/routers/user.py

```diff
@@ -16,11 +17,12 @@ def get_all_users():
         response = []
         for row in result:
             (id, name, createdAt) = row
-            response.append({ "id": id, "name": name, "createdAt": createdAt })
-        return { "users": response }
+            response.append({"id": id, "name": name, "createdAt": createdAt})
+        return {"users": response}
+
 
 @router.get("/{user}")
-async def get_user(user: str):
+async def get_user(user: int):
     try:
         user = int(user)
     except Exception:
```

Comment:
should be able to remove this try-except block... I think FastAPI does the type validation internally, so if an int isnt given then it will try to parse it or throw an error